src/sqlfluff/core/rules/config_info.py

```diff
@@ -65,7 +65,7 @@
         "definition": (
             "The capitalisation policy to enforce, extended with PascalCase. "
             "This is separate from capitalisation_policy as it should not be "
-            "applied to keywords."
+            "applied to keywords"
```

Comment:
@barrywhart question about this one. I think we should keep the full stops here and remove it from whatever template it's getting subbed into.

Reason is that some config descriptions end with question marks (e.g. `multiline_newline`) so we have a weird `?.` in the docs:
![image](https://user-images.githubusercontent.com/80432516/148138723-0cd5e1ad-a759-4ecb-bcdd-723adfa7e87f.png)

Reply:
Will do. I found the place and will update it.